libminifi/src/io/ZlibStream.cpp

```diff
@@ -127,7 +143,12 @@ ZlibDecompressStream::ZlibDecompressStream(gsl::not_null<OutputStream*> output,
 
 ZlibDecompressStream::~ZlibDecompressStream() {
   if (state_ != ZlibStreamState::UNINITIALIZED) {
-    inflateEnd(&strm_);
+    int result = inflateEnd(&strm_);
+    if (result == Z_STREAM_ERROR) {
+      logger_->log_debug("Stream state was inconsistent");
+    } else if (result != Z_OK) {
+      logger_->log_debug("Unknown error while finishing decompression %d", result);
```

Comment:
Is it only a debug level information? Seems like a quite bad state for me.

Reply:
changed to `log_error`